crates/pallet-domains/src/lib.rs

```diff
@@ -810,14 +809,18 @@ mod pallet {
             origin: OriginFor<T>,
             runtime_name: Vec<u8>,
             runtime_type: RuntimeType,
```

Comment:
Now I'm wondering why do we need these two fields, why would generic `pallet-domains` need to care that there exists EVM domain at all? Name also seens unnecessary on the surface.